blocks/cards/cards.js

```diff
@@ -1,6 +1,15 @@
 import { createOptimizedPicture, readBlockConfig, lookupPages } from '../../scripts/scripts.js';
 
-function decorateChampionCards(champions, block) {
+function decorateChampionCards(block) {
+  [...block.children].forEach((row) => {
+    const children = row.querySelectorAll('div');
+    const pars = children[1].querySelectorAll('p');
+    pars[0].classList.add('cards-card-bubble');
+    pars[1].classList.add('cards-card-country');
```

Comment:
this "country" field is optional (does not appear on sentry) and breaks if it's not there. i've got both examples in drafts to compare:
* [sentry](https://champion-cards-curated--theplayers--hlxsites.hlx.page/drafts/fkakatie/sentry-past-champions)
* [wgc](https://champion-cards-curated--theplayers--hlxsites.hlx.page/drafts/fkakatie/wgc-past-champions)